### PR TITLE
FR-1569 - adding consent order for contested event and fields and permissions

### PR DESCRIPTION
**Pull request by ocarleton**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FR-1569

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/contested/json/CaseEventToFields/CaseEventToFields-generalOrder-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,216 @@
+[
```
Comment (shaunganley) [inline definitions/contested/json/CaseEventToFields/CaseEventToFields-generalOrder-nonprod.json:1]: is there a reason this file is called general order when the rest are consent order?

File: definitions/contested/json/CaseEvent/CaseEvent-consentOrder-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[
```
Comment (shaunganley) [inline definitions/contested/json/CaseEvent/CaseEvent-consentOrder-nonprod.json:1]: couldn't we get away with putting everything in the prod file except the caseEventAuthorisation file? would give us less to clean up at the end

File: definitions/contested/json/State.json [hunk 3]
```diff
@@ -182,5 +182,15 @@
...
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "consentedOrderSubmitted",
+    "Name": "Consented Order Submitted",
+    "DisplayOrder": 24,
+    "TitleDisplay": "**${[CASE_REFERENCE]}  ${applicantLName}  vs ${respondentLName}**"
   }
+
+
```
Comment (shaunganley) [inline definitions/contested/json/State.json:195]:
```suggestion
```

File: definitions/contested/json/CaseEvent/CaseEvent-consentOrder-nonprod.json (hunk 2 above)
Comment (ocarleton) [reply definitions/contested/json/CaseEvent/CaseEvent-consentOrder-nonprod.json]: Very good point, I'll do this now

**Pull request merged**